catalog-info.yaml

```diff
@@ -155,3 +155,54 @@ spec:
         everyone:
           access_level: BUILD_AND_READ
 
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/e57ee3bed7a6f73077a3f55a38e76e40ec87a7cf/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: buildkite-pipeline-elastic-package-test-serverless
+  description: Pipeline to test elastic-package with Serverless projects
+  links:
+    - title: Pipeline
+      url: https://buildkite.com/elastic/elastic-package-test-serverless
+
+spec:
+  type: buildkite-pipeline
+  owner: group:ingest-fp
+  system: buildkite
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: elastic-package-test-serverless
```

Comment:
Is this pipeline name ok? do you prefer to add something else into this name ?

Reply:
It is fine.